app/controllers/api/v1/groups_controller.rb

```diff
@@ -40,6 +45,19 @@ def add_user
     head :no_content
   end
 
+  def list_admins
+    group = Group.find(params[:id])
+    users = group.group_admins.joins(:user).
```

Comment:
can we also check if users in `group_admins` are not inactive. and if inactive users are there can we filter them out?